config/jobs/kubernetes/sig-aws/eks/presubmits.yaml

```diff
@@ -0,0 +1,36 @@
+
+
+presubmits:
+  # Run Kubernetes 1.10 branch e2e tests with EKS prod build 1.10
```

Comment:
this job doesn't specify a branch, though, FWIW. 
Also, we don't normally support CI for long once a kubernetes version is unsupported, 1.10 will be going out of support with 1.13 out (1.13, 1.12, 1.11) As part of setting up the 1.14 release jobs the 1.10 jobs will be removed.